src/test/java/MailActionsVerifyTest.java

```diff
@@ -0,0 +1,41 @@
+import base.TestBase;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import utility.TestListener;
+
+@Listeners(TestListener.class)
+public class MailActionsVerifyTest extends TestBase {
+
+    private final String USER_1_LOGIN = "seleniumtests10";
+    private final String USER_2LOGIN = "seleniumtests30";
+    private final String MAIL_INDEX = "@gmail.com";
+    private final String USER_PASSWORD = "060788avavav";
+
+
+    @Test(description = "GM-3 Verify the ability to send emails")
+    public void sendMailAbilityCheck() {
+        loginPage.login(USER_1_LOGIN, USER_PASSWORD)
```

Comment:
I see that you use flow pattern. But you are making important error in all tests. Your sequences are infinite, and it's not clear at all where test is now. Please add variables every time, when you are on new page. Example:
```
HomePage homePage = loginPage.login(..., ....);
NExtPage nextPage = homePage.clickSomething()
                            .clickSOmethind2();
```
Also my opinion is that classic Page Object is better.